searxng_extra/docs_prebuild

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# lint: pylint
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+"""Script that implements some prebuild tasks needed by target docs.prebuild
+"""
+
+import sys
+import os.path
+import time
+from searx import settings, get_setting, doc
+
+_doc_user = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'docs', 'user'))
+
+def main():
+
+    DOC = None
+    base_url = get_setting('server.base_url', None)
+
+    if base_url:
+        DOC = _render_all_with_flask_ctx(base_url)
```

Comment:
Do you plan to use this?
In which context?
If on, let's drop the support for this feature (if base_url is not None, then the URL are rendered in the Sphinx documentation).